cluster/saltbase/salt/kube-addons/kube-addons.sh

```diff
@@ -19,6 +19,39 @@
 # managed result is of that. Start everything below that directory.
 KUBECTL=/usr/local/bin/kubectl
 
+function create-kubernetesauth-secret() {
```

Comment:
Blah, I didn't enforce it well in `create-object`, but you can fix it while you're in here. :)

These should all be `local`. Probably `local -r token=$1`, etc, since they look readonly.